ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1117,6 +1117,51 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-merge-robot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  decoration_config:
```

Comment:
can you add the rehearsal label?

Reply:
I don't think that we want to rehearse that. The rehearse job will open a new PR.